src/main/java/persistence/common/FieldValue.java

```diff
@@ -0,0 +1,26 @@
+package persistence.common;
+
+import java.lang.reflect.Field;
+
+public class FieldValue {
+    private FieldClazz fieldClazz;
+    private String value;
```

Comment:
```suggestion
public class FieldValue<T> {
    private FieldClazz<T> fieldClazz;
    private T value;
```

기왕 FieldClazz 안에 clazz 멤버가 있는데 generic 을 더 적극적으로 활용해보는 것은 어떨지 제안을 해봅니다.